CONTRIBUTING.md

```diff
@@ -61,3 +61,5 @@ TomaszLukawskiSam
 
 * PR can be merged by PR owner or TAU reviewer if at least one TAU Reviewer approved and verified it.
 * TAU Reviewers can not approve their own PR.
+* When merging **standard PR use "Rebase and merge"** option in order to have straight git history
+* When merging changes between tau_* branches use "Create a merge commit" option in order to allow to synchronization between branches.
```

Comment:
What's the "Create a merge commit" option?
Could you explain it in more detail?

Reply:
This is standard merge, similar to git merge command. It merges histories of two branches.
It is good when synchronizing branches with each other (e.g. tau_1.0 <--> tau_1.1 <--> tau_1.2)
When we have common history between branches, git is able to automatically put changes from one branch to the second one with git merge command.
If we decide to use "rebase" option instead of merge in the next synchronization we will have problems because rebased commits common to both branches will not be recognized.

I think the core differences are nicely described here:
https://www.atlassian.com/git/tutorials/merging-vs-rebasing